theme.js

```diff
@@ -978,6 +978,13 @@ export default () => ({
       },
     },
 
+  '.inline': {
+    container: {
+      width: window.width,
+        position: 'relative',
```

Comment:
Please fix indentation. Remove two extra spaces. Also, have you tested it how does it work in situations like this:
```javascript
render() {
  return (
    <Screen>
      <NavigationBar />
      <ScrollView>
        {/*Some scrollable content*/}
      </ScrollView>
    </Screen>
  );
}
```

I want to know would scroll view's content overlapse navigation bar after scroll... If it would, then add zIndex to navigation bar's style.

Also, could you add this styleName to the documentation as well?